src/docs/ab-testing/index.mdx

```diff
@@ -0,0 +1,135 @@
+Our A/B testing framework is built on [Facebook's planout library](http://facebook.github.io/planout/).
+
+Let's say you want to optimize the clickthrough rate for Sentry links and decide to test whether to show Plugins in the Integration Directory.
+
+For any experiment, there are 2 events of interest:
+
+- Exposure: when the user is subjected to the experiment. In this case it is when a link is shown.
+- Action: the behavior we're trying to observe, which should always be tied to a metric. In this case the action is a click and the metric is the clickthrough rate.
+
+## Adding an A/B Test
+
+To launch the A/B test you will need to do the following:
+
+1. Set up an A/B test in Getsentry to place users/organizations into different test groups.
+2. Adding the experiment to the list of experiments in Sentry for Typescript definitions
+3. Add the logic where you have to control the in-app behavior
+
+### Changes to `getsentry/getsentry`
+
+In **getsentry/getsentry**, locate the  `getsentry/experiments/config.py` file). In this file, you will define an experiment class that inherits from either `OrgExperiment` or `UserExperiment`. Choose one depending on whether you want to assign the experiment across each user or each organization.
+
+For this example, we will use the `OrgExperiment` , this way everyone in the organization will be given the same experiment assignment.
+
+```python
+# In getsentry/experiments/config.py
+
+from __future__ import absolute_import
+
+from django.conf import settings
+
+from .base import OrgExperiment
+from planout.ops.random import WeightedChoice
+
+WEIGHT_VARIANT = 0.5
+
+class ShowPluginsExperiment(OrgExperiment):
+    def default_value(self):
+        return "0"
+
+    def assign(self, params, org, actor):
+        if settings.IS_TEST or org.slug == "sentry":
+            params.variant = "1"
+        else:
+            params.variant = WeightedChoice(
+                choices=["0", "1"], weights=[1 - WEIGHT_VARIANT, WEIGHT_VARIANT], unit=org.id
+            )
+```
+
+A few things
+
+- You can set the assignment value to whatever is necessary for your experiment.
+- Assignment logic: The `assign` function on the experiment class determines which group an entity falls into. In the most simple case, you have just two groups: a control and test group. If that's the case, you can use `BernoulliTrial` to generate the test value. If you have three or more groups, you can use `UniformChoice`. You can also insert whatever custom logic you want, such as enabling an experiment if a particular feature flag is set.
+- You can use other random assignment operators. Learn more: [https://facebook.github.io/planout/docs/random-operators.html](https://facebook.github.io/planout/docs/random-operators.html)
+
+We are setting the values to "0" or "1" and giving a 50% chance an organization sees the plugins in the Integration Directory
+
+Initialize the experiment in `getsentry/web/apps.py` :
+
+```python
+class Config(AppConfig):
+		. 
```

Comment:
This looks kinda strange on the page imo, do you think this was meant to be `...`? 

<img width="763" alt="Screenshot 2023-05-04 at 3 42 50 PM" src="https://user-images.githubusercontent.com/29959063/236344949-8b52b5e9-18c1-4255-9ef2-430bd4a13e52.png">